src/road_cmd.cpp

```diff
@@ -2404,7 +2404,7 @@ CommandCost CmdConvertRoad(TileIndex tile, DoCommandFlag flags, uint32 p1, uint3
 
 			if (flags & DC_EXEC) { // we can safely convert, too
 				/* Update the company infrastructure counters. */
-				if (!IsRoadStopTile(tile) && CanConvertRoadType(owner, rtt) && owner != OWNER_TOWN) {
+				if (owner == _current_company) {
```

Comment:
is removing the `!IsRoadStopTile` check a good idea here?

Reply:
Ping @ddm999, can you take a look at this again?